ads_plugin.py

```diff
@@ -529,7 +524,7 @@ def on_save_segmentation_button(self, event):
         axon_array = axon_array * params.intensity['binary']
 
 
-        image_name = myelin_mask_overlay.name[:-len("_seg-myelin")]
+        image_name = myelin_mask_overlay.name[:-len("_seg-myelin-manual_pred")]
```

Comment:
I'm not sure what this is, the other suffix in the ads_plugin don't have the "seg" part.